lib/timetable-form/Form.tsx

```diff
@@ -1,36 +1,57 @@
-import { useState } from "react";
-import "govuk-frontend/dist/govuk/govuk-frontend.min.css";
+import { useMemo, useState } from "react";
+import { v4 as uuidv4 } from "uuid";
 
 import { Button } from "../gds-components/button/Button";
 import { TextInput } from "../gds-components/text-input/TextInput";
 import { DateInput } from "../gds-components/date-input/DateInput";
-import { Stages, FormData } from "../types/timetable";
-import {
-  formStateToDevelopmentPlanTimetables,
-  devPlanToCSVString,
-} from "../utils/timetable";
+import { DevelopmentPlan } from "../types/timetable";
+import { devPlanToCSVString } from "../utils/timetable";
 import { stageNames } from "../constants";
 
 import styles from "./styles.module.css";
+import "govuk-frontend/dist/govuk/govuk-frontend.min.css";
+import { PlanPreview } from "../timetable-visualisation/PlanPreview";
 
 const defaultDate = "";
 
-const initialStages = Object.fromEntries(
-  stageNames.map((stage) => [stage, defaultDate])
-) as Stages;
+const initialState: DevelopmentPlan = {
```

Comment:
This is long and repeated can we do something else? Maybe make the type a class and use a default value? Up to you

Reply:
you are right this is also in Visualisation component. I will move it to the constants file :)